lib/rows.js

```diff
@@ -244,6 +242,8 @@ function queryRowsOperationImpl(self, builtPlan, streamType, options, bindingArg
       endpoint += (complexValues === 'reference') ? sep + 'node-columns=reference' : '';
       break;
   }
+  // Specify database if defined in client connection
```

Comment:
This comment looks obsolete now and should be removed.

Reply:
I can remove this and also rename "initialize" in internal.js but giving review comments together will save time in the future for both reviewer and developer.